chain/client/tests/process_blocks.rs

```diff
@@ -68,7 +67,6 @@ pub fn create_nightshade_runtimes(genesis: &Genesis, n: usize) -> Vec<Arc<dyn Ru
 /// Runs block producing client and stops after network mock received two blocks.
 #[test]
 fn produce_two_blocks() {
-    init_test_logger();
```

Comment:
Why removing test_logger everywhere?

Reply:
It actually prevented me from seeing which test was failing due to the amount of log output